### PR TITLE
fix: minor design fixes

### PR DESCRIPTION
**Pull request by nerdyspook**
> Fixed Homepage styes
> Added animations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,13 +93,13 @@ <h1 class="hero__title">
 
             <section class="info">
                 <div class="sale">
-                    <img src="./assets/images/hero-shoe.jpg" alt="" />
+                    <img src="./assets/images/sale.jpg" alt="" />
```
Comment (KanekarSnehal) [inline index.html]: Alt tag is empty

**Review by KanekarSnehal — APPROVED** [review]
No issues found, you can merge it.

File: index.html (hunk 1 above)
Comment (nerdyspook) [reply index.html]: Thanks, I have made the changes

**Review by MaruthiPothuganti — COMMENTED** [review]
The code looks well structured and UI looks pretty Awesome

**Pull request merged**